### PR TITLE
Added data-x libraries

### PR DESCRIPTION
**Pull request by afo**
Data-X: Tried adding DL libraries (TF + Keras are most important).

### CODE REVIEW

File: user-image/requirements.txt [hunk 1]
```diff
@@ -46,3 +46,18 @@ pymc3==3.2
...
 corner==2.0.1
 #
+#
+# data-x DL
+#
+# TensorFlow
+tensorflow>=1.3.0
+# Image
+pillow
+scikit-image
```
Comment (yuvipanda) [inline user-image/requirements.txt]: Can you put explicit versions on all these?

**Comment by yuvipanda** [comment]
@henchc do you have the time to take this one through to deployment?

File: user-image/requirements.txt (hunk 1 above)
Comment (henchc) [reply user-image/requirements.txt]: if your install works and are the versions you want you can run `pip freeze` to see the versions

**Comment by henchc** [comment]
@afo can you commit to your fork:

```
# TensorFlow
tensorflow==1.3.0
# Image processing
Pillow==4.3.0
scikit-image==0.13.1
# Keras-related
Theano==0.9.0
Keras==2.0.8
keras-vis==0.4.1
# HDF5
tables==3.4.2
h5py==2.7.1
```

**Pull request closed without merging**